src/cmap/auth/scram.ts

```diff
@@ -2,7 +2,7 @@ import * as crypto from 'crypto';
 import { promisify } from 'util';
 
 import { Binary, type Document } from '../../bson';
-import { saslprep } from '../../deps';
+import * as deps from '../../deps';
```

Comment:
What's the reason behind this change?

Reply:
Slack: Needed for mocking, b/c otherwise this file gets a reference to saslprep before sinon can stub it.

Maybe it's worth leaving a comment that the import is intentionally different than our others.